src/backend/aspen/database/models/pathogens.py

```diff
@@ -27,6 +27,10 @@ class Pathogen(idbase):  # type: ignore
         comment=("full pathogen abbreviated name, ex: SARS-CoV-2"),
     )
 
+    samples = relationship("Sample", back_populates="pathogen")
+    phylo_runs = relationship("PhyloRun", back_populates="pathogen")
```

Comment:
I don't think we need to set up these relationships in both directions? Just setting up the relationships in the Samples/Trees/Runs tables should make these backrefs work.

Reply:
i thought it might be nice to setup this so that we could pull all the related samples, phylotrees, phyloruns for a given pathogen easily? For example I feel like Pathogen.samples might be nice and make that query easy?

Reply:
Yeah, but that's handled by the `back_populates` parameter of the relationship in the other models!

Reply:
errr  --- maybe it's backref:
https://docs.sqlalchemy.org/en/14/orm/relationship_api.html#sqlalchemy.orm.relationship.params.backref

Reply:
OK, TIL! 

We've [been using `backref`](https://docs.sqlalchemy.org/en/14/orm/relationship_api.html#sqlalchemy.orm.relationship.params.backref) in most of our relationships, which populates relationships in both directions. It's super convenient, but the docs say that current best practice is to use `back_populates` in both directions since it plays nicer with type checking. 

This looks good then, I think we'll have to start migrating our old `backref` parameters at some point.

Reply:
lol yeah your docs reading was way more up to date than mine was!! 🥇 😍